bark/runtime/viewer/pygame_viewer.py

```diff
@@ -210,6 +210,10 @@ def clear(self):
         self.screen.fill(self.background_color)
         self.screen_surface.fill(self.background_color)
 
+    def getSizeOfColormap(self):
+        # TODO
+        return 1
```

Comment:
is this intended to be merged into the master?

Reply:
The color coding for the matplotlib viewer got improved. 

so i figured that for the sake of completeness, I add the function to the pygame viewer as well, as it already has this:

```

    def getColorFromMap(self, double_color):
        # TODO
        return (1.0, 0, 0)
```